force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -731,6 +736,14 @@ export default class GeFormRenderer extends LightningElement{
             if (!canUpdate) { return; }
         }
 
+        const recordId = this.formState['Id'];
+        if (recordId && this.selectedPaymentMethod() !== PAYMENT_METHOD_CREDIT_CARD) {
+            const gift = this.giftBatch.findGiftBy(recordId);
```

Comment:
I think an alternative to instantiating a geGiftBatch to find the individual gift would be to instantiate the geGift directly using giftInView.

Reply:
That makes a lot of sense to me. Would it be ideal to instantiate the geGift when we initially load the individual gift into form state?